augur/types.py

```diff
@@ -19,6 +20,22 @@ def __str__(self) -> str:
         """
         return self.value
 
+    @classmethod
+    def argtype(cls, input_string):
+        """
+        Intended to be used as the argument type converter for argparse options
+        that use the enum values as inputs.
+
+        Raises a custom `argparse.ArgumentTypeError` so that the error
+        message can include a helpful list of the all valid enum values.
+        """
+        try:
+            return cls(input_string)
+        except ValueError as error:
+            choices = ", ".join(map(lambda x: f"'{str(x)}'", cls))
```

Comment:
_non-blocking_

```suggestion
            choices = ", ".join(f"{str(x)!r}" for x in cls)
```

Or similar? Main point here is to use `repr()` (here via `!r`) instead of manually surrounding with single quotes.

Reply:
Thanks! Updated in the forced-push.